crates/biome_js_analyze/src/lint/nursery/no_restricted_imports.rs

```diff
@@ -36,14 +41,24 @@ declare_rule! {
     }
 }
 
+/// Options for each path of rule `noRestrictedImports`.
+#[derive(Clone, Debug, Default, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct RestrictedModuleConfig {
+    pub message: String,
+    pub allowed_from: Option<Vec<String>>,
```

Comment:
No need for `Option` here. An empty `Vec` is fine as a default.

Reply:
I've used the Option just because in my IDE when i don't use it it appears a warning message on the JSON:

![image](https://github.com/biomejs/biome/assets/5855426/9dec34c5-2751-47ec-9474-f44bffff8ec9)